js/ui/file_manager/ui.file_manager.files_tree_view.js

```diff
@@ -122,39 +126,19 @@ class FileManagerFilesTreeView extends Widget {
         }
     }
 
-    _onModelSelectedItemLoaded(item) {
-        if(this._filesTreeView) {
-            this._handleModelSelectedItemLoaded(item);
-        } else {
-            setTimeout(() => this._handleModelSelectedItemLoaded(item));
-        }
-    }
-
-    _handleModelSelectedItemLoaded(item) {
-        const $element = this._getItemElement(item);
-        this._changeSelectedItem(item, $element);
-    }
-
-    _changeSelectedItem(item, $element) {
-        this._setSelectedItem(item, $element);
-        this._raiseCurrentFolderChanged();
-    }
-
-    _setSelectedItem(item, $element) {
-        this._selectedItem = item || null;
-        this._updateFocusedElement($element);
-    }
-
-    _updateFocusedElement($element) {
+    _updateFocusedElement() {
+        const directoryInfo = this._getCurrentDirectory();
+        const $element = this._getItemElementByKey(directoryInfo.fileItem.key);
         if(this._$focusedElement) {
             this._$focusedElement.toggleClass(FILE_MANAGER_DIRS_TREE_FOCUSED_ITEM_CLASS, false);
         }
         this._$focusedElement = $element || $();
+        this._$focusedElement.dirInfo = directoryInfo;
```

Comment:
Can we use .data() function here? For example:
`this._$focusedElement.data("dirInfo", directoryInfo);`